src/custom/state/enhancedTransactions/reducer.ts

```diff
@@ -43,72 +48,91 @@ export const initialState: EnhancedTransactionState = {}
 
 const now = () => new Date().getTime()
 
-export default createReducer(initialState, (builder) =>
-  builder
-    .addCase(
-      addTransaction,
-      (transactions, { payload: { chainId, from, hash, hashType, approval, summary, presign } }) => {
-        if (transactions[chainId]?.[hash]) {
-          throw Error('Attempted to add existing transaction.')
+export default createReducer(
+  initialState,
+  (builder) =>
+    builder
+      .addCase(
+        addTransaction,
+        (transactions, { payload: { chainId, from, hash, hashType, approval, summary, presign, safeTransaction } }) => {
+          if (transactions[chainId]?.[hash]) {
+            throw Error('Attempted to add existing transaction.')
```

Comment:
throwing in reducers is a side-effect:
https://github.com/reduxjs/redux/issues/1960#issuecomment-248025505

i would avoid this

Reply:
Agree, will try to remove. It was in the original code anyways. Maybe I will make a noop and log something...